eventuate-core/src/main/scala/com/rbmhtechnology/eventuate/log/EventLog.scala

```diff
@@ -395,7 +395,7 @@ abstract class EventLog[A <: EventLogState](id: String) extends Actor with Event
       remoteReplicationProgress += targetLogId -> (0L max from - 1)
       replicationRead(from, clock.sequenceNr, max, scanLimit, evt => evt.replicable(currentTargetVersionVector, filter)) onComplete {
         case Success(r) => self.tell(ReplicationReadSuccess(r.events, from, r.to, targetLogId, null), sdr)
-        case Failure(e) => self.tell(ReplicationReadFailure(e.getMessage, targetLogId), sdr)
+        case Failure(e) => self.tell(ReplicationReadFailure(e, targetLogId), sdr)
```

Comment:
I wonder if this is a bit risky?!? The exception can very well be backend specific, right? It gets serialized with standard java serialization and it might not be on the classpath of the receiving application which would make deserialization impossible, right?